src/views/pages/RevokeKey/index.js

```diff
@@ -1,10 +1,29 @@
 import React from 'react';
```

Comment:
page errors without this import

```suggestion
import React, {useState} from 'react';
```